backup.sh

```diff
@@ -70,38 +73,47 @@ do
     echo "Packing up results to $ARCHIVE_FILENAME"
     if [ $retval -eq 0 ]; then
         echo "Successfully packed backup into $ARCHIVE_FILENAME"
-        mv "$ARCHIVE_FILENAME" "$TARGETDIR"
+        cp "$ARCHIVE_FILENAME" "$TARGETDIR"
     else
         echo "Failed creating $ARCHIVE_FILENAME"
     fi
 
     rm -rf $FILES
   else
     # Move files from intermediate work to target directory
-    echo "Move backup files to target directory"
-    find $WORKDIR -type f -name "*.$CURRENT_DB.bak" -exec mv {} $TARGETDIR \;
-    find $WORKDIR -type f -name "*.$CURRENT_DB.trn" -exec mv {} $TARGETDIR \;
+    echo "Copy backup files to target directory"
+    find $WORKDIR -type f -name "*.$CURRENT_DB.bak"     -exec cp {} $TARGETDIR \;
+    find $WORKDIR -type f -name "*.$CURRENT_DB.trn"     -exec cp {} $TARGETDIR \;
+  fi
+
+  # Push to remote directory
+  if [ "$PUSH_REMOTE_MODE" = "move" ] || [ "$PUSH_REMOTE_MODE" = "copy" ]; then
+    echo "Push backup to remote directory"
+    find $WORKDIR -type f -name "*.$CURRENT_DB.*" -exec cp {} $REMOTEDIR \;
+
+    if [ "$PUSH_REMOTE_MODE" = "move" ]; then
+      echo "Cleanup target directory"
+      find $TARGETDIR -type f -name "*.$CURRENT_DB.*" -exec rm {} \;
+    fi
   fi
 
-  # Delete intermediate directory
-  echo "Delete intermediate directory"
+  # Cleanup intermediate directory
+  echo "Cleanup intermediate directory"
+  find $WORKDIR -type f -name "*.$CURRENT_DB.*" -exec rm {} \;
```

Comment:
Couldn't we just `rm -rf` instead of deleting the files and than using `rmdir`

Reply:
@pfeigl done.